manifests/crds/kustomization.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- https://github.com/fluxcd/kustomize-controller/releases/download/v0.11.0/kustomize-controller.crds.yaml
-- https://github.com/fluxcd/source-controller/releases/download/v0.11.0/source-controller.crds.yaml
-- https://github.com/fluxcd/helm-controller/releases/download/v0.9.0/helm-controller.crds.yaml
-- https://github.com/fluxcd/notification-controller/releases/download/v0.12.0/notification-controller.crds.yaml
-- https://github.com/fluxcd/image-reflector-controller/releases/download/v0.8.0/image-reflector-controller.crds.yaml
-- https://github.com/fluxcd/image-automation-controller/releases/download/v0.8.0/image-automation-controller.crds.yaml
+- https://github.com/fluxcd/source-controller/releases/download/v0.12.0/source-controller.crds.yaml
```

Comment:
This should probably be added to the automation magic.

Reply:
@hiddeco see https://github.com/fluxcd/flux2/issues/1258 😉

Reply:
I'll now perform this exercise while we wait for CI jobs.